src/main/java/stringcalculator/domain/Number.java

```diff
@@ -0,0 +1,29 @@
+package stringcalculator.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Pattern;
+
+public class Number {
+
+    private final List<String> userInput;
+
+    public Number(List<String> userInput) {
+        this.userInput = userInput;
```

Comment:
Number 보다는 `Numbers`의 의미가 더 적절한 것 같아요. 🤔
또한 여기서는 input 값을 그대로 Stirng 리스트로 가지고 있는 것 보다는 Number(int형)로 변경해보면 어떨까요? 😃
지금의 역할은 단순한 `Validator` 역할로 밖에 안보여요. 🥲

아래와 같이 변경해보면 어떨까요? 🤔
```suggestion
public class Numbers {

    private final List<Integer> numbers;

    public Number(List<String> numbers) {
        this.numbers = numbers.stream()
                .map(Integer::new)
                .collect(Collectors.toList());
```

또한 Numbers 객체가 Calculator가 하고있는, `calculate` 기능을 수행하도록 해도 좋을 것 같아요. 😉

Reply:
넵 말씀해주신 부분 반영했습니다 
감사합니다 :)